Sources/Commands/SwiftTestTool.swift

```diff
@@ -9,7 +9,8 @@ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
 import class Foundation.ProcessInfo
-
+import class Foundation.FileManager
+import struct Foundation.URL
```

Comment:
We should not be using `FileManager`, but instead the file system / path APIs in Basic, e.g. https://github.com/apple/swift-package-manager/blob/master/Sources/Basic/FileSystem.swift#L140

Reply:
done

Reply:
I think the URL import is now unused?

Reply:
done